pkg/apis/servicecatalog/types.go

```diff
@@ -343,6 +355,15 @@ type ServiceInstanceSpec struct {
 	//
 	// Immutable.
 	ExternalID string
+
+	// Currently, this field is ALPHA: it may change or disappear at any time
+	// and its data will not be migrated.
+	//
+	// UserInfo contains information about the user that last modified this
+	// instance. This field is set by the API server and not settable by the
+	// end-user. User-provided values for this field are not saved.
+	// +optional
+	UserInfo *UserInfo
```

Comment:
Any reason we're not putting this into Status?

Reply:
We will put it into status in a future PR where we put fields in status for 'what the controller is doing now' and 'what does the broker know about'

Reply:
okie dokie. We should probably do a review of all resources before beta to move things between the Spec and Status.

Reply:
Yeah, I agree with a full review of everything before we move to v1beta1 version of API